docs/Technical API/Sensor/readme.md

```diff
@@ -0,0 +1,58 @@
+# Sensor class
+
+`Sensor`s are procedures to access external resources and procedures. For example, reading from raw data, feature engineering staffs, and preprocessing procedures.
```

Comment:
Can we say: Sensors are backbox functions mostly written by the application programmer that can help for pre-processing, feature extraction, and obtaining representations based on external resources, for example getting the lemma of a word or Glove representation of a word.